File: 프로그래머스/0/181920. 카운트 업/카운트 업.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public List<Integer> solution(int start_num, int end_num) {
        List<Integer> answer = new ArrayList<>();
        
        // start_num부터 end_num까지 숫자를 차례로 리스트에 추가
        for (int i = start_num; i <= end_num; i++) {
            answer.add(i);
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181920. 카운트 업/카운트 업.kt
class Solution {
    fun solution(start_num: Int, end_num: Int): IntArray {
        // start_num부터 end_num까지 숫자를 리스트로 만들고 IntArray로 변환
        return (start_num..end_num).toList().toIntArray()
    }
}
